cmd/reporter/reporter.go

```diff
@@ -29,14 +29,15 @@ import (
 )
 
 var (
-	configFile = flag.String("config-file", "", "required report config file to use")
-	walkPath   = flag.String("walk-path", "", "path to search for Walks")
-	reviewFile = flag.String("review-file", "", "path to the file containing a list of last-known-good states - this needs to be writeable")
-	hostname   = flag.String("hostname", "", "host to review the differences for")
-	beforeFile = flag.String("before-file", "", "path to the file to compare against (last known good typically)")
-	afterFile  = flag.String("after-file", "", "path to the file to compare with the before state")
-	paginate   = flag.Bool("paginate", false, "pipe output into $PAGER in order to paginate and make reviews easier")
-	verbose    = flag.Bool("verbose", false, "print additional output for each file which changed")
+	configFile     = flag.String("config-file", "", "required report config file to use")
+	walkPath       = flag.String("walk-path", "", "path to search for Walks")
+	reviewFile     = flag.String("review-file", "", "path to the file containing a list of last-known-good states - this needs to be writeable")
+	hostname       = flag.String("hostname", "", "host to review the differences for")
+	beforeFile     = flag.String("before-file", "", "path to the file to compare against (last known good typically)")
+	afterFile      = flag.String("after-file", "", "path to the file to compare with the before state")
+	paginate       = flag.Bool("paginate", false, "pipe output into $PAGER in order to paginate and make reviews easier")
+	verbose        = flag.Bool("verbose", false, "print additional output for each file which changed")
+	noReviewUpdate = flag.Bool("no-review-update", false, "don't ask to update the \"last known good\" review")
```

Comment:
Sometimes double negation is harder to understand. It's clear in the flag description here but in the code below `if !*noReviewUpdate` might be a bit confusing. How about `skip-review-update` ? It's only 2 characters longer.

Or the opposite: `update-review` flag wich defaults to `true`. Then one can `-update-review=false`.
Alternatively, make it a breaking change and have `update-review` default to `false`. Then one needs to explicitely say `-update-review=true`. I guess it depends on which mode is used more frequently. Our team would use `-update-review=false` most of the time. OTOH we run it from a script which always sets appropriate flags.

We've just made a breaking change with flags name anyway if you're worried about backwards compatibility.

I'm also fine with leaving this as is if you think it's best.

Reply:
I like doing the opposite and having it false by default if that's fine.